### ISSUE TITLE
<Playground /> doesn't like '$'

### ISSUE THREAD

**Issue by kentcdodds** [pedronauck/docz#201]
## Bug Report

**Describe the bug**

Source:

```javascript
<Playground>
  Playground is fine with the $ character by itself
</Playground>
```

Result:

<img width="642" alt="screen shot 2018-08-08 at 2 30 55 pm" src="https://user-images.githubusercontent.com/1500684/43862742-cf1f9ee4-9b17-11e8-8b32-8ca94a48b9f1.png">

---

Source:

```javascript
<Playground>
  Playground does not like the '$' character in single quotes
</Playground>
```

Result:

<img width="647" alt="screen shot 2018-08-08 at 2 31 04 pm" src="https://user-images.githubusercontent.com/1500684/43862727-c1803fbe-9b17-11e8-896f-257783f14025.png">

**Expected behavior**

I expect that the playground result for the $ in single quotes will render properly, but it does not.

My guess is there's some regex magic going on and the source isn't being properly escaped.

**Comment by rakannimer** [comment]
This also happens with backticks. 

```javascript
<Playground>
`This doesnt work`
 <MyComponent didMount={()=>{
	alert(`This also doesnt work`)
}}
</Playground>
```

Looks like the issue is [here](https://github.com/pedronauck/docz/blob/7a84638aa140888b0102885372bcacb9c797c395/packages/rehype-docz/src/index.ts#L39)

I'm working on a fix, should have a PR soon 😄

**Comment by pedronauck** [comment]
Fixed on the new [v0.10.0](https://github.com/pedronauck/docz/releases/tag/v0.10.0) 🚀

**Comment by kentcdodds** [comment]
Thank you!
